src/Components/Logo/InfluxDataLogo.tsx

```diff
@@ -0,0 +1,119 @@
+// Libraries
+import React, {forwardRef} from 'react'
+import classnames from 'classnames'
+
+// Types
+import {StandardFunctionProps, InfluxColors} from '../../Types'
+
+// Styles
+import './InfluxDataLogo.scss'
+
+export interface InfluxDataLogoProps extends StandardFunctionProps {
+  /** Coloration of the SVG image */
+  fill?: InfluxColors | string
+}
+
+export type InfluxDataLogoRef = SVGSVGElement
```

Comment:
SVGElement

Reply:
If I use `SVGElement` it requires `href` and 45 other attributes, and doesn't like `ref`. For some reason `SVGSVGElement` seems to be the correct type for inline svgs + refs